stylesheets/landing.css

```diff
@@ -0,0 +1,210 @@
+/* This stylesheet will contain all the styling for the landing page */
+
+/* Project was designed with a mobile first approach */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0 auto;
+  overflow-x: hidden;
+}
+
+h1 {
+  font-family: 'Crete Round', serif;
+}
+
+p {
+  font-family: 'Roboto', sans-serif;
+}
+
+/* actual page styling */
+
+/* Header section */
+
+.header {
+  height: 92px;
+  width: 375px;
+  left: 0;
+  top: 0;
+  border-radius: 0;
+  background-color: #3c3a39;
+}
+
+.topHeaderBar {
+  margin: 0 10px 10px 10px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 10px;
+  color: #fff;
+}
+
+.authorheader h2 {
+  font-size: 52px;
+  font-family: 'Crete Round', serif;
+  font-weight: normal;
+  font-style: normal;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  color: #ff6800;
+}
```

Comment:
- [OPTIONAL]

- [ ] I can see that the correct `font-weight` in the Figma template is `400`. please consider adding the correct font weight so that your design matches that of your chosen template.